lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -3,7 +3,6 @@
 import java.time.Duration;
```

Comment:
This entire file was just refactored and "technically" has nothing to do with this PR. I just stumpled over it and had the time to fix it :)